client/asset/eth/nodeclient_harness_test.go

```diff
@@ -1996,7 +2087,7 @@ func testRefund(t *testing.T, assetID uint32) {
 			t.Fatalf("%s: unexpected pending in balance %d", test.name, in)
 		}
 
-		if err := waitForMined(test.refunderClient, 10, false); err != nil {
+		if err := waitForMined(10, false); err != nil {
```

Comment:
There's an ignored error from `test.refunderContractor.value` just above.  I'm not sure if that can even error in this test or not.

Reply:
No longer ignoring.